libs/leonsans/src/leonsans.ts

```diff
@@ -0,0 +1,608 @@
+/*!
+ * VERSION: 1.6.3
+ * DATE: 2019-09-13
+ * https://leon-sans.com
+ *
+ * @license Copyright (c) 2019, Jongmin Kim. All rights reserved.
+ **/
+import { Graphics } from 'pixi.js';
```

Comment:
```suggestion
import PIXI from 'pixi.js';
```
`PIXI.SomeClassOrType` 꼴로 사용하는 경우가 있는 것 같은데, 네임스페이스가 헷갈리지 않게 전체 import를 해서 사용하는 건 어떠신가요??